### ISSUE TITLE
dsp_vector_abs often outputs negative numbers

### ISSUE THREAD

**Issue by oscarbailey-xmos** [xmos/lib_dsp#121]
e.g.   
```
    int32_t vec1[1] = {1932};
    int32_t vec_result[1];
    dsp_vector_abs(vec1, vec_result, 1);
    // vec_result = {-1932};
```

**Comment by andrewstanfordjason** [comment]
this should be fixed

**Comment by mbanth** [comment]
Fixed by Pull Request #199.

**Comment by mbanth** [comment]
Nightly runs of various libraries that depend on this one have completed successfully.  On that basis, I have closed the issue.
